README.md

```diff
@@ -337,8 +337,9 @@ Currently, the following package managers are supported:
 * [NPM](https://www.npmjs.com/) (Node.js)
 * [NuGet](https://www.nuget.org/) (.NET, with currently some
   [limitations](https://github.com/oss-review-toolkit/ort/pull/1303#issue-253860146))
-* [PIP](https://pip.pypa.io/) (Python)
-* [Pipenv](https://pipenv.readthedocs.io/) (Python)
+* [PIP](https://pip.pypa.io/) (Python, currently limited to projects that are compatible with Python 2.7 or Python 3.6)
+* [Pipenv](https://pipenv.readthedocs.io/) (Python, currently limited to projects that are compatible with Python 2.7 or
+  Python 3.6)
```

Comment:
Maybe a link to the ORT issue would be helpful here, so potential users can find the discussion about this issue.

Reply:
Done.